### PR TITLE
Added function and test for Choose in numpy

### PR DESCRIPTION
**Pull request by SubbuS-22**

### CODE REVIEW

File: ivy/functional/frontends/numpy/indexing_routines/indexing_like_operations.py [hunk 1]
```diff
@@ -80,3 +80,27 @@ def fill_diagonal(a, val, wrap=False):
     a = ivy.reshape(a, a.size)
     a[:end:step] = val
     a = ivy.reshape(a, shape)
+
+
+@to_ivy_arrays_and_back
+def choose(indexes, choices, mode='raise'):
```
Comment (hirwa-nshuti) [inline ivy/functional/frontends/numpy/indexing_routines/indexing_like_operations.py:86]: The function signature is supposed to be
```python
def choose(a, choices, out=None, mode='raise'):
    pass
```

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_indexing_routines/test_indexing_like_operations.py [hunk 2]
```diff
@@ -272,3 +272,36 @@ def test_numpy_fill_diagonal(
...
+)
+def test_numpy_choose(
+    *,
+    dtype_x_indices_axis,
+    test_flags,
+    frontend,
+    fn_tree,
+    on_device,
+):
+    dtypes, choices, indices, _ = dtype_x_indices_axis
```
Comment (hirwa-nshuti) [inline ivy_tests/test_ivy/test_frontends/test_numpy/test_indexing_routines/test_indexing_like_operations.py:298]: There is an error in unpacking the generated test examples as it is a tuple of 5 elements

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Hey @SubbuS-22 

I didn't go deep into the implementation but here are first things that need to be taken care of.
Do check the comments below.
Thanks🙂

File: ivy/functional/frontends/numpy/indexing_routines/indexing_like_operations.py (hunk 1 above)
Comment (hirwa-nshuti) [reply ivy/functional/frontends/numpy/indexing_routines/indexing_like_operations.py:86]: Also ivy doesn't have the `choose` function

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.

**Pull request closed without merging**